cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -118,6 +118,8 @@ def test_service_generic_catalog_bundle(appliance, catalog_item):
     assert provision_request.is_succeeded(), msg
 
 
+@pytest.mark.uncollectif(lambda appliance: appliance.version > '5.9',
```

Comment:
Optional: You can use `@pytest.mark.ignore_stream("5.10")` if you don't want to collect this at all for 5.10 version.